### PR TITLE
chore(Styles): Add text selection mixin

### PR DESCRIPTION
**Pull request by FlyersPh9**
These components use `::selection`:
- Alert
- Labeled inputs
- Progress indicator
- Table
- Toast notification

This PR adds `@mixin iui-text-selection;` so the opacity is always in sync across components.

### CODE REVIEW

File: src/style/mixins.scss [hunk 1]
```diff
@@ -63,6 +63,15 @@
   }
 }
 
+@mixin iui-text-selection($status: primary) {
+  &::selection,
```
Comment (veekeys) [inline src/style/mixins.scss:67]: Whats the difference between these 2 selectors?

**Review by veekeys — APPROVED** [review]
Apart the facts that PR came from nowhere and the problem stated in description does not exist (all values were the same, so it shouldnt have been inconsistent); opacity was slightly changed and we dont have visual tests to test all of that (I guess too minor); PR looks ok. 

PR title could not consist `feat`, because I dont think this change requires minor version release...

File: src/style/mixins.scss (hunk 1 above)
Comment (mayank99) [reply src/style/mixins.scss:67]: `&` will apply to the same level where the mixin is used, `*` will apply to all descendants in the tree

File: src/style/mixins.scss [hunk 2]
```diff
@@ -63,6 +63,15 @@
...
+  *::selection {
+    @include themed {
+      background-color: rgba(t(iui-color-foreground-#{$status}-rgb), t(iui-opacity-5));
```
Comment (mayank99) [inline src/style/mixins.scss:70]: Shouldn't this be `opacity-4` like in every other place?

**Comment by FlyersPh9** [comment]
> Apart the facts that PR came from nowhere

This came from the work taking place in #505.

> and the problem stated in description does not exist (all values were the same, so it shouldnt have been inconsistent)

I changed the description after I submitted because I realized what had happened.  I changed the opacity in `global.scss` but did not change it in all the other places.  It took me a while before I figured out what had happened.

> opacity was slightly changed

I've reverted it back to 4 for this PR but I will change it to 5 in #505.

> PR looks ok.

Thank you. ☺️ 

> PR title could not consist `feat`, because I dont think this change requires minor version release...

Changed to `chore` after reverting opacity back to 4.

File: src/style/mixins.scss (hunk 2 above)
Comment (mayank99) [reply src/style/mixins.scss]: Maybe make opacity a parameter in the mixin then

Comment (mayank99) [reply src/style/mixins.scss]: or well, lets see what we need in #505 and change accordingly

**Review by mayank99 — APPROVED** [review]

**Pull request merged**